### PR TITLE
add lpg_planner

### PR DESCRIPTION
**Pull request by h-kamada**
DO NOT MERGE

### CODE REVIEW

**Comment by k-okada** [comment]
sorry  I have reverted this merge, please create this PR again, 
http://web.mit.edu/larsb/Public/16.412/pset%204/LPGdownloads/README-LPG1.2.txt says it shulde be  lpg1.2-linux.tar.gz or lpg1.2-unix.tar.gz

File: 3rdparty/lpg_planner/package.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<package>
+  <name>lpg_planner</name>
+  <version>2.0.14</version>
+  <description>lpg_planner: LPGL Planner (http://eracle.ing.unibs.it/lpg/)</description>
+  <maintainer email="h-kamada@jsk.imi.i.u-tokyo.ac.jp">Yohei Kakiuchi</maintainer>
```
Comment (k-okada) [inline 3rdparty/lpg_planner/package.xml]: please fix Yohei Kakiuchi to Hitoshi Hamada

**Comment by k-okada** [comment]
please contact one of
"Current members: Alfonso E. Gerevini , Alessandro Saetti, Ivan Serina, and Mauro Vallati (PhD student) "
(http://prometeo.ing.unibs.it/lpg/) and ask for fixing web server

**Comment by h-kamada** [comment]
OK, I sent e-mail to the current members.

**Pull request merged**